TrackletAlgorithm/ProjectionRouterBuffer.h

```diff
@@ -41,16 +42,18 @@ class ProjectionRouterBufferBase<DISK,AllProjectionType>// : public VMProjection
 public:
   enum BitWidths {
     // Bit sizes for ProjectionRouterBufferMemory fields
-    kPRBufferIsPSSeedSize = 0,
+    kPRBufferIsPSSeedSize = 1,
     //kPRProjSize = VMProjection<VMPTYPE>::kVMProjectionSize,
     kPRBufferZBinSize = MEBinsBits+1+1,
     kPRBufferNStubsSize = 4*kNBits_MemAddrBinned,
     kPRBufferMaskStubsSize = 4,
     kPRBufferIndexSize = 7,
-    kPRBufferPhiProjBinSize = 1,
+    kPRBufferShiftSize = 2,
```

Comment:
Curious as to why this is needed?

Reply:
Same as above. It is a signed number, so we need 2 bits.